### ISSUE TITLE
[token-cli] Remove deprecated functions from `solana-clap-v3-utils`

### ISSUE THREAD

**Issue by samkim-crypto** [solana-labs/solana-program-library#6393]
#### Problem
The PR https://github.com/solana-labs/solana-program-library/pull/6376 upgrades token-cli to use clap-v3. However, it still uses some functions from `solana-clap-v3-utils` that are deprecated.

#### Solution
Remove and clean up these deprecated functions.

**Comment by tonton-sol** [comment]
the token-cli has not yet been updated to v3

**Comment by samkim-crypto** [comment]
Yeah, the PR https://github.com/solana-labs/solana-program-library/pull/6376 should do it, but it will probably take a while for it to get reviewed and merged. The PR has to make a whole bunch of changes at once in order for it to compile and so it is tedious and time-consuming to review all the changes, unfortunately.
